use/developer/api-documentation/bot-setup-apis.md

```diff
@@ -105,51 +199,69 @@ For any bot we will have to specify a certain configuration which is a part of c
---data-raw '{
+--header 'Accept: application/json, text/plain, */*' \
+--header 'ownerOrgID: org01' \
+--header 'ownerID: 8f7ee860-0163-4229-9d2a-01cef53145ba' \
+--data '{
     "data": {
-        "name": "UCI demo bot logic",
+        "name": "TEST",
+        "description": "TEST",
         "transformers": [
             {
-                "id": "bbf56981-b8c9-40e9-8067-468c2c753659",
+                "id": "774cd134-6657-4688-85f6-6338e2323dde",
                 "meta": {
                     "form": "https://hosted.my.form.here.com",
-                    "formID": "List-Button-test-v1"
+                    "formID": "testing_form",
+                    "title": "TEST",
+                    "body": "TEST",
+                    "serviceClass": "SurveyService",
+                    "hiddenFields": [
+                        {
+                            "name": "mobilePhone",
+                            "path": "mobilePhone",
+                            "type": "param",
+                            "config": {
+                                "dataObjName": "user"
+                            }
+                        }
+                    ],
+                    "templateType": "JS_TEMPLATE_LITERALS",
+                    "type": "broadcast"
                 }
             }
         ],
-        "adapter": "44a9df72-3d7a-4ece-94c5-98cf26307323"
+        "adapter": "6efa8087-0939-49ab-b8e5-5676e036c17b"
     }
 }'
 ```
 
-**Response**: It will return a conversation logic id, use it in create bot api. Eg. **92f7b965-4118-4ddc-9c7d-0bc0f77092db**
+**Response**: It will return a conversation logic id, use it in create bot. Eg. **be37c9f2-7f2b-4e19-b525-33b0a2aabdd5**
 
 ```
 {
-    "ts": "2022-05-24T13:48:06.407Z",
-    "params": {
-        "resmsgid": "23b94970-db68-11ec-ae84-fbd67a9c1174",
-        "msgid": null,
-        "status": "successful",
-        "err": null,
-        "errmsg": null
-    },
-    "responseCode": "OK",
+    "apiId": "api.admin.conversationLogic",
+    "path": "/admin/conversationLogic",
+    "msgid": "df8edb08-f887-41cd-8f7a-604d3229dabc",
     "result": {
-        "data": {
-            "transformers": "[{"id":"bbf56981-b8c9-40e9-8067-468c2c753659","meta":{"form":"https://hosted.my.form.here.com/%22,/%22formID/%22:/%22List-Button-test-v1/%22%7D%7D]",
-            "adapter": "44a9df72-3d7a-4ece-94c5-98cf26307323",
-            "name": "UCI demo bot logic",
-            "id": "92f7b965-4118-4ddc-9c7d-0bc0f77092db"
-        }
-    }
+        "id": "be37c9f2-7f2b-4e19-b525-33b0a2aabdd5",
+        "name": "TEST",
+        "createdAt": "2023-07-05T05:29:57.310Z",
+        "updatedAt": "2023-07-05T05:29:57.311Z",
+        "description": null,
+        "adapterId": "6efa8087-0939-49ab-b8e5-5676e036c17b"
+    },
+    "startTime": "2023-07-05T05:29:57.293Z",
+    "method": "POST",
+    "ownerId": "8f7ee860-0163-4229-9d2a-01cef53145ba",
+    "endTime": "2023-07-05T05:29:57.314Z"
 }
 ```
 
-### 2.4 Add a bot
+### 2.5 Add a bot
```

Comment:
The flow should be similar to the documentation shared with Dell. 

In order to conversate with users the following steps need to be taken: 
* Create a conversation bot
    * Upload ODK Form
    * Add a Conversation Logic
    * Configure a bot
* Create a broadcast bot (optional)
    * Configure a Conversation Logic with similar name as 1 to link both
    * Add User Segment
    * Add Bot for Notification and send notification to all user segments